discovery-provider/plugins/notifications/src/sns.ts

```diff
@@ -28,7 +30,10 @@ export const publish = async (params: PublishCommandInput) => {
 export const publishBatch = async (params: PublishBatchCommandInput) => {
   try {
     const data = await snsClient.send(new PublishBatchCommand(params))
-    console.log("Success.", data)
+    // TODO remove?
+    console.log("sns batch send success")
+    // console.log("Success.", data)
+    // console.log("Success.", data)
```

Comment:
why is there a todo remove?
I would remove the log

Reply:
Added a todo for the log that I added. Can remove both later, for now it's useful for testing on staging